# try:
    # Statement which could generate
    # Exception
# except:
      # Solution of generated exception
# finally:
      # block of code which is going to
      # execute in any situation

____________________________________________________________________________________________________________________________________________________________________

# a = input("Enter the number: ")
# print(f"Multiplication table of {a} is: ")

# try:
#     for i in range(1, 11):
#         print(f"{int(a)} X {i} = {int(a) * i}")
# except Exception as e:
#     print("Invalid Input")

# print("Some imp lines of code")
# print("End of program")

# try:
#    num = int(input("Enter an integer: "))
#    print("You've entered an integer value.")
 #except ValueError:
#    print("Number entered is not an integer.")








